neuralmonkey/learning_utils.py

```diff
@@ -209,6 +209,13 @@ def training_loop(tf_manager: TensorFlowManager,
                                         tf_manager.best_score_batch),
                                 color='blue')
 
+                            # store also graph parts
+                            all_coders = set.union(
+                                *[rnr.all_coders for rnr in runners])
+                            for coder in all_coders:
+                                for session in tf_manager.sessions:
+                                    coder.save(session)
```

Comment:
nejsem si jistej, jestli to chcem ukládat furt, @jlibovicky k tomu možná něco řikal a já to zapomněl. 

Navíc, jak je to tady teď, tak se to bude ukládat fakt furt, ne jen když je nejlepší model.